x/rollup/types/msgs.go

```diff
@@ -74,9 +74,39 @@ func (m *MsgInitiateWithdrawal) ValidateBasic() error {
 }
 
 func (*MsgInitiateWithdrawal) Type() string {
-	return "l2withdrawal"
+	return "l2_eth_withdrawal"
 }
 
 func (*MsgInitiateWithdrawal) Route() string {
 	return ModuleName
 }
+
+var _ sdktypes.Msg = (*MsgInitiateERC20Withdrawal)(nil)
+
+func (m *MsgInitiateERC20Withdrawal) ValidateBasic() error {
```

Comment:
~Do you know where these `ValidateBasic`s are called?~ [Never mind](https://github.com/cosmos/cosmos-sdk/blob/8bfcf554275c1efbb42666cc8510d2da139b67fa/baseapp/baseapp.go#L879)